WordPress/src/main/res/values/strings.xml

```diff
@@ -2747,6 +2747,7 @@
     <string name="prepublishing_nudges_tags_action">Tags</string>
     <string name="prepublishing_nudges_home_header_publishing_to">Publishing to</string>
     <string name="prepublishing_nudges_home_publish_button">Publish Now</string>
+    <string name="prepublishing_nudges_home_schedule_button">Schedule Now</string>
```

Comment:
Thanks. This makes sense.

Reply:
@malinajirka I have a question about this because I want to do this the right way. So I just did a check and realized that there are 4 strings with `Publish Now`. Should I follow the example you have shared above and just replace those ones as well? Somewhat outside the scope but always good to leave the code better than we saw it right? :)

Reply:
It's important to check that all the strings are used in the same context. As the fact that one word can be used in a different context in English doesn't mean it can be used in a different context in other languages.
I believe "Publish Now" is used in the same context, so I think it'd be a nice clean-up ;).

Reply:
> It's important to check that all the strings are used in the same context. As the fact that one word can be used in a different context in English doesn't mean it can be used in a different context in other languages.

So true! 

> I believe "Publish Now" is used in the same context, so I think it'd be a nice clean-up ;).

Great!

Reply:
Resolved here 2d07a7b